### PR TITLE
Add test default GloFAS station test

### PR DESCRIPTION
**Pull request by Piotrk39**
## Describe your changes

Adds test for GloFAS station to be visible by default in no trigger flood.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review

### CODE REVIEW

**Review by jannisvisser — CHANGES_REQUESTED** [review]
Looks good! Just some small notes.

File: tests/e2e/tests/Map/GloFAStaionsVisibleNoTrigger.spec.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  await page.goto('/');
+  await loginPage.login(
+    NoTriggerDataSet.UserMail,
+    NoTriggerDataSet.UserPassword,
+  );
+});
+
+test(
+  qase(
+    28,
```
[the remark this answers is not in the document]
Comment (jannisvisser) [reply tests/e2e/tests/Map/GloFAStaionsVisibleNoTrigger.spec.ts:40]: @Piotrk39 this is the thing I was asking about in Sprint Review. Maybe quick call to discuss?

File: tests/e2e/tests/Map/GloFAStaionsVisibleNoTrigger.spec.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  ),
+  async ({ page }) => {
+    const dashboard = new DashboardPage(page);
+    const userState = new UserStateComponent(page);
+    const map = new MapComponent(page);
+
+    // Navigate to disaster type the data was mocked for
+    await dashboard.navigateToFloodDisasterType();
+    // Assertions
+    await userState.headerComponentIsVisible({
```
[the remark this answers is not in the document]
Comment (Piotrk39) [reply tests/e2e/tests/Map/GloFAStaionsVisibleNoTrigger.spec.ts:51]: It makes sure we don't have a timeout. it is a small check that lasts less then 100ms but with a big batch of tests it plays a role.

File: tests/e2e/Pages/MapComponent.ts [hunk 3]
```diff
@@ -172,5 +174,18 @@ class MapComponent extends DashboardPage {
...
+  async gloFASMarkersAreVisible() {
+    // Wait for the page to load
+    await this.page.waitForSelector('[alt="glofas-station-marker"]');
+
+    // Count the number of gloFAS markers
+    const gloFASMarkersCount = await this.gloFASMarker.count();
+    const nthSelector = this.getRandomInt(1, gloFASMarkersCount);
+
+    // Assert that the number of gloFAS markers is greater than 0 and randomly select one to be visible
+    expect(gloFASMarkersCount).toBeGreaterThan(0);
```
[the remark this answers is not in the document]
Comment (Piotrk39) [reply tests/e2e/Pages/MapComponent.ts:187]: This is actually a great idea and I'll keep it. I searched for similar solution for the districts outline check I will make a note of that. For now checking for at least one visible marker is more then enough. We could do more precise scenario for a trigger state if you think it is a good idea @jannisvisser ?

File: tests/e2e/tests/Map/GloFAStaionsVisibleNoTrigger.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Piotrk39) [reply tests/e2e/tests/Map/GloFAStaionsVisibleNoTrigger.spec.ts:40]: Generally speaking any disaster type could have been picked up for this initial batch. More or less willingly those tests are dependent on the disaster type if there is a specific marker to be checked. In this case it is a bout GloFAS so "Flood" disaster type is a given setting. In other cases it is not important at all and I could come back to those scenarios and review them. From what was discussed flood is for now a go to type but the main focus is on the state of the trigger. Starting from "No-trigger" and now following with a "Trigger". With a resource allocation and the time I kind of believe it is as high level as possible tho if you see improvements here I am open for suggestions.

File: tests/e2e/tests/Map/GloFAStaionsVisibleNoTrigger.spec.ts
[the remark this answers is not in the document]
Comment (Piotrk39) [reply tests/e2e/tests/Map/GloFAStaionsVisibleNoTrigger.spec.ts:1]: Fixed. I guess it was about missing "s"

File: tests/e2e/tests/Map/GloFAStaionsVisibleNoTrigger.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (jannisvisser) [reply tests/e2e/tests/Map/GloFAStaionsVisibleNoTrigger.spec.ts]: Ok. good enough for now. I guess we will run into the need to clear things up, when we actually expand to other disaster types. We can make the clarifications then.

File: tests/e2e/Pages/MapComponent.ts [hunk 4]
```diff
@@ -172,5 +174,18 @@ class MapComponent extends DashboardPage {
     expect(redCrossMarkersCount).toBeGreaterThan(0);
     await expect(this.redCrossMarker.nth(nthSelector)).toBeVisible();
   }
+
+  async gloFASMarkersAreVisible() {
```
[the remark this answers is not in the document]
Comment (Piotrk39) [reply tests/e2e/Pages/MapComponent.ts:204]: I know now exactly what was you question. It is fixed now. Checks both legend and layer and in the end the map

**Review by jannisvisser — APPROVED** [review]
assuming the test passes, approved

**Pull request merged**